webapp/src/css/private-vars.less

```diff
@@ -0,0 +1,29 @@
+/**
+* WARNING:
+* Don't import this file. This is only for "variable.less" usage.
+* Don't use these variables. The variable names aren't descriptive enough.
+* Instead, use the variables from "variable.less".
+*/
+@white: #FFFFFF;
```

Comment:
This variable is still used in https://github.com/medic/cht-core/blob/7862-private-color-variables/webapp/src/css/content-list.less#L225 but I don't know if that's intended

Reply:
Yes, this is intended, that piece changed with the [bulk delete work](https://github.com/medic/cht-core/pull/7846/files#diff-7531bbe609e35f2936eac69d48cd591e18105b79dce25dc64de3c5c4833c1418L224) it was my attempt to reduce code conflicts :)